views/partials/myLibraryContent.handlebars

```diff
@@ -0,0 +1,29 @@
+{{!-- views/myLibrary.handlebars --}}
```

Comment:
This is a partial that's supposed to template for reuse the user's library; may need some work or not needed.